app/src/main/res/layout/activity_main.xml

```diff
@@ -1,49 +1,135 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/relativeLayout"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:orientation="vertical"
```

Comment:
you are not using this Constraint layout, you can delete it, also this propertie `android:orientation="vertical"` correspond to the LinearLayout, not Contraint

Reply:
Done